src/styles/theme/themes.ts

```diff
@@ -1,16 +1,34 @@
 import { IDefaultThemeProps, IBreakpoints, IThemeProps } from "./";
 
-const breakpoints: IBreakpoints = {
+const breakpoints = {
   breakpoints: {
-    extraSmall: "368px",
-    small: "576px",
-    medium: "768px",
-    large: "998px",
-    extraLarge: "1200px",
-    big: "1400px",
+    mobileS: "320px",
+    mobileM: "375px",
+    mobileL: "425px",
+    tablet: "768px",
+    laptop: "1024px",
+    desktop: "1440px",
+    desktopL: "2560px",
   },
 };
 
+const devices = {
+  devices: {
+    mobileS: `(min-width: ${breakpoints.breakpoints.mobileS})`,
+    mobileM: `(min-width: ${breakpoints.breakpoints.mobileM})`,
+    mobileL: `(min-width: ${breakpoints.breakpoints.mobileL})`,
+    tablet: `(min-width: ${breakpoints.breakpoints.tablet})`,
+    laptop: `(min-width: ${breakpoints.breakpoints.laptop})`,
+    desktop: `(min-width: ${breakpoints.breakpoints.desktop})`,
+    desktopL: `(min-width: ${breakpoints.breakpoints.desktopL})`,
+  },
+};
```

Comment:
Transforma este em um objeto mais simples

```suggestion
const devices = {
    mobileS: `(min-width: ${breakpoints.mobileS})`,
    mobileM: `(min-width: ${breakpoints.mobileM})`,
    mobileL: `(min-width: ${breakpoints.mobileL})`,
    tablet: `(min-width: ${breakpoints.tablet})`,
    laptop: `(min-width: ${breakpoints.laptop})`,
    desktop: `(min-width: ${breakpoints.desktop})`,
    desktopL: `(min-width: ${breakpoints.desktopL})`,
};
```